geowebcache/restng/src/main/java/org/geowebcache/restng/controller/GWCController.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author Arne Kepp / The Open Planning Project 2009
+ * @author David Vick, Boundless, Copyright 2017
+ *
+ * Original file
+ * https://github.com/GeoWebCache/geowebcache/blob/master/geowebcache/rest/src/main/java/org/geowebcache/rest/GWCRestlet.java
```

Comment:
Lets us know the name of the original file, but the link is going to break when we get rid of the old rest module in master.  That it it includes an indication that there's an original file to look for and what it's named is enough for me though.